### DISCUSSION TITLE
pull-local deletes the commits from source repo

### DISCUSSION THREAD

**Discussion by roshan20039** [ostreedev/ostree#3324] [Q&A]
I have two repos repo1 with one commit (h1) and repo2 with 3 commits(h2,h3,h4). When I do ostree --repo=repo1 --depth=-1 pull-local repo2 branch and do ostree --repo=repo1 log branch, it has h2,h3 and h4 and h1 is not there. 
It is expected that h1 should also be there right? 
The log output displays << History beyond this commit not fetched >> as well.
Could someone help me understand this?

**Comment by quic-roshs** [comment]
Hi,

Any help on this query?
